viewing_party/party.py

```diff
@@ -1,23 +1,128 @@
 # ------------- WAVE 1 --------------------
 
+from multiprocessing.sharedctypes import Value
+from re import I
+
+from tests.test_constants import GENRE_1, MOVIE_TITLE_1, RATING_1
+
+
 def create_movie(title, genre, rating):
-    pass
+    
+    new_movie = {}
+    if title and genre and rating:
+        new_movie = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+        }
+        return new_movie
+    return None

```

Comment:
Let's consider flipping the sense of the condition around to handle the special case (something is falsy) quickly and get out, rather than having that dangling else or return statement at the end. This arrangement is referred to as a guard clause or a guard condition. 

Notice that it lets us move the main focus of the function (creating the new dict) out of an indented block, letting it stand out more clearly.

```suggestion
    if not title and not genre and not rating:
        return None
    new_movie = {
        "title" : title,
        "genre" : genre,
        "rating" : rating
    }
    return new_movie

```
It may seem like a silly change or nitpicky, but this is common structuring in Python when using guard clauses.